### DISCUSSION TITLE
api/connections does not show client provided name

### DISCUSSION THREAD

**Discussion by pabermod** [rabbitmq/rabbitmq-server#5817] [Other] [answered]
`api/connections` does not show client provided name when metrics collector is disabled.

But `rabbitmqctl list_connections client_properties` shows it...

Is this intended? If this is the case I think it should be documented

**Comment by michaelklishin** [comment, marked as answer]
This is the first time someone has noticed the difference in a few years since client-provided name support was introduced.

If you disable metric collection, some or all metrics may go missing.

CLI tools inspect connections themselves, not a set of metrics they emit, hence the difference.
